[2024-07-22T02:36:00.894+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: example_dynamic_task_mapping.add_one scheduled__2022-12-24T00:00:00+00:00 map_index=0 [queued]>
[2024-07-22T02:36:00.921+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: example_dynamic_task_mapping.add_one scheduled__2022-12-24T00:00:00+00:00 map_index=0 [queued]>
[2024-07-22T02:36:00.924+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-07-22T02:36:00.929+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2024-07-22T02:36:00.933+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-07-22T02:36:01.022+0000] {taskinstance.py:1300} INFO - Executing <Mapped(_PythonDecoratedOperator): add_one> on 2022-12-24 00:00:00+00:00
[2024-07-22T02:36:01.029+0000] {standard_task_runner.py:55} INFO - Started process 3287 to run task
[2024-07-22T02:36:01.042+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'example_dynamic_task_mapping', 'add_one', 'scheduled__2022-12-24T00:00:00+00:00', '--job-id', '1205', '--raw', '--subdir', '/home/***/.local/lib/python3.7/site-packages/***/example_dags/example_dynamic_task_mapping.py', '--cfg-path', '/tmp/tmpcyjy7j_2', '--map-index', '0']
[2024-07-22T02:36:01.046+0000] {standard_task_runner.py:83} INFO - Job 1205: Subtask add_one
[2024-07-22T02:36:01.179+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 1205 for task add_one ([Errno 2] No such file or directory: '/opt/airflow/logs/dag_id=example_dynamic_task_mapping/run_id=scheduled__2022-12-24T00:00:00+00:00/task_id=add_one/map_index=0/attempt=1.log'; 3287)
[2024-07-22T02:36:01.222+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2024-07-22T02:36:01.388+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
